e2e/cypress/integration/integrations/incoming_webhook/basic_formatting_spec.js

```diff
@@ -0,0 +1,126 @@
+
+            // * Verify that the BOT label appears
+            cy.get('.Badge').should('be.visible').and('have.text', 'BOT');
+
+            // * Verify that there's no status indicator
+            cy.get('.status').should('not.exist');
+
+            // # Verify that the elements on posted message matched as expected
+            cy.get('.post-message__text').within(() => {
+                cy.wrap(el).should('contain', 'The following escaped characters should appear normally');
+                cy.wrap(el).should('contain', '(ampersand, open angle, close angle): & < >');
+                cy.wrap(el).should('contain', 'The following should appear as links:');
+                cy.get('.markdown__link').eq(0).
+                    should('have.text', 'This is a link to about-dot-mattermost-dot-com').
+                    and('have.attr', 'href', 'https://about.mattermost.com/');
+                cy.get('.markdown__link').eq(1).
+                    should('have.text', 'Markdown Link also to About page').
+                    and('have.attr', 'href', 'https://about.mattermost.com/');
+                cy.wrap(el).should('contain', 'Normal Link:');
+                cy.get('.markdown__link').eq(2).
+                    should('have.text', 'https://about.mattermost.com/').
+                    and('have.attr', 'href', 'https://about.mattermost.com/');
+                cy.wrap(el).should('contain', 'Mail Link:');
+                cy.get('.markdown__link').eq(3).
+                    should('have.text', 'Email').
+                    and('have.attr', 'href', 'mailto:mail@example.com');
+                cy.wrap(el).should('contain', 'The following should be markdown formatted');
+                cy.wrap(el).should('contain', '(mouse emoji, strawberry emoji, then formatting as indicated):');
+                cy.get('.emoticon').eq(0).parent().
+                    should('have.html', '<span alt=":hamster:" class="emoticon" title=":hamster:" style="background-image: url(&quot;http://localhost:8065/static/emoji/1f439.png&quot;);">:hamster:</span>');
+                cy.get('.emoticon').eq(1).parent().
+                    should('have.html', '<span alt=":strawberry:" class="emoticon" title=":strawberry:" style="background-image: url(&quot;http://localhost:8065/static/emoji/1f353.png&quot;);">:strawberry:</span>');
+                cy.wrap(el).find('strong').should('have.text', 'bold');
+                cy.wrap(el).find('em').should('have.text', 'italic');
+                cy.wrap(el).find('strong').should('have.text', 'bold');
+                cy.get('.codespan__pre-wrap').should('have.html', '<code>code</code>');
+                cy.wrap(el).find('del').should('have.text', 'strike');
+                cy.get('.mention-link').eq(0).should('have.text', '#hashtag');
+                cy.get('.mention-link').eq(1).should('have.text', `@${otherUser.username}`);
```

Comment:
This is a candidate for visual testing but for the meantime, let's have it this way.